composer.json

```diff
@@ -9,36 +9,16 @@
         "project-base/app/",
         "project-base/src/Shopsys/"
       ],
-      "Shopsys\\FormTypesBundle\\": [
-        "packages/form-types-bundle/src/"
-      ],
-      "Shopsys\\HttpSmokeTesting\\": [
-        "packages/http-smoke-testing/src/"
-      ],
-      "Shopsys\\MigrationBundle\\": [
-        "packages/migrations/src/Shopsys/MigrationBundle/"
-      ],
-      "Shopsys\\Plugin\\": [
-        "packages/plugin-interface/src/"
-      ],
-      "Shopsys\\ProductFeed\\": [
-        "packages/product-feed-interface/src/"
-      ],
-      "Shopsys\\ProductFeed\\ZboziBundle\\": [
-        "packages/product-feed-zbozi/src/"
-      ],
-      "Shopsys\\ProductFeed\\GoogleBundle\\": [
-        "packages/product-feed-google/src/"
-      ],
-      "Shopsys\\ProductFeed\\HeurekaBundle\\": [
-        "packages/product-feed-heureka/src/"
-      ],
-      "Shopsys\\ProductFeed\\HeurekaDeliveryBundle\\": [
-        "packages/product-feed-heureka-delivery/src/"
-      ],
-      "Shopsys\\FrameworkBundle\\": [
-        "packages/framework/src/"
-      ]
+      "Shopsys\\FormTypesBundle\\": "packages/form-types-bundle/src/",
+      "Shopsys\\HttpSmokeTesting\\": "packages/http-smoke-testing/src/",
+      "Shopsys\\MigrationBundle\\": "packages/migrations/src/Shopsys/MigrationBundle/",
```

Comment:
Btw, now I see this directory structure is different to all others. Why is that?

I'd expect this:

```json
   "Shopsys\\MigrationBundle\\": "packages/migrations/src/",
```

Reply:
I see. What is the reason to keep it this way? 
I think [monorepo-tools](https://github.com/shopsys/monorepo-tools) handles this easily